controllers/vsl.go

```diff
@@ -122,41 +122,38 @@ func (r *DPAReconciler) ValidateVolumeSnapshotLocations(dpa oadpv1alpha1.DataPro
 			Spec: *vslSpec.Velero,
 		}
 
+		// check if VSL name is specified in DPA spec
```

Comment:
Here, lets delete `vsl` variable (it was used only by `EventRecorder`) and use same approach as the other function in this file does to get VSL name

https://github.com/openshift/oadp-operator/blob/fb2411e2948874a6abe61ca31b1cd9957621ebb6/controllers/vsl.go#L227-L231

Reply:
ack